Sources/StreamChat/Query/MessageReminderListQuery.swift

```diff
@@ -0,0 +1,130 @@
+//
+// Copyright © 2025 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// A namespace for the `FilterKey`s suitable to be used for `MessageReminderListQuery`. This scope is not aware of any extra data types.
+public protocol AnyMessageReminderListFilterScope {}
+
+/// An extra-data-specific namespace for the `FilterKey`s suitable to be used for `MessageReminderListQuery`.
+public struct MessageReminderListFilterScope: FilterScope, AnyMessageReminderListFilterScope {}
+
+/// Filter keys for message reminder list.
+public extension FilterKey where Scope: AnyMessageReminderListFilterScope {
+    /// A filter key for matching the `channel_cid` value.
+    /// Supported operators: `in`, `equal`
+    static var cid: FilterKey<Scope, ChannelId> { .init(
+        rawValue: "channel_cid",
+        keyPathString: #keyPath(MessageReminderDTO.channel.cid),
+        valueMapper: { $0.rawValue }
+    ) }
+
+    /// A filter key for matching the `message_id` value.
+    /// Supported operators: `in`, `equal`
+    static var messageId: FilterKey<Scope, MessageId> { .init(rawValue: "message_id", keyPathString: #keyPath(MessageReminderDTO.id)) }
+
+    /// A filter key for matching the `remind_at` value.
+    /// Supported operators: `equal`, `greaterThan`, `lessThan`, `greaterOrEqual`, `lessOrEqual`
+    static var remindAt: FilterKey<Scope, Date> { .init(rawValue: "remind_at", keyPathString: #keyPath(MessageReminderDTO.remindAt)) }
+
+    /// A filter key for matching the `created_at` value.
+    /// Supported operators: `equal`, `greaterThan`, `lessThan`, `greaterOrEqual`, `lessOrEqual`
+    static var createdAt: FilterKey<Scope, Date> { .init(rawValue: "created_at", keyPathString: #keyPath(MessageReminderDTO.createdAt)) }
+}
+
+public extension Filter where Scope: AnyMessageReminderListFilterScope {
```

Comment:
how did you come up with these filters?

Reply:
It is in the Notion's Backend Specs

Reply:
Ah, or do you mean the helpers? If you mean the helpers, it is because it will be the most common filters, so I think it should be useful for customers